.gitignore

```diff
@@ -32,4 +32,5 @@ build/
 ### VS Code ###
 .vscode/
 #properties
-*.properties
\ No newline at end of file
+*.properties
+src/main/resources/application.properties
```

Comment:
Redundancy

You only need `*.properties`, git should already handle all the cases for you.

Reply:
```*.propertites``` has added. However, the application.propertites file still appeared in git's change.